guides/security-oidc-okta-openshift.adoc

```diff
@@ -0,0 +1,245 @@
+= Securing WildFly Applications Using Okta on OpenShift
+:summary: A step by step guide on how to secure a WildFly application using Okta's OpenID Provider and deploy it to OpenShift. 
+:includedir: _includes
+include::{includedir}/_attributes.adoc[]
+:prerequisites-time: 15
+
+WildFly applications can be secured using OpenID Connect (OIDC) and deployed to OpenShift. By using OIDC to secure applications, you delegate authentication to OIDC providers. The `elytron-oidc-client` subsystem can be used to secure an application deployed to WildFly using any OpenID Provider. This guide demonstrates how to secure an example application deployed to WildFly on OpenShift using https://www.okta.com/[Okta] as the OpenID Provider. 
+
+include::{includedir}/_prerequisites.adoc[]
+* Roughly 15 minutes
+* Access to an OpenShift cluster (try the https://developers.redhat.com/developer-sandbox[Red Hat Developer Sandbox] for free)
+* https://docs.openshift.com/container-platform/4.14/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI]
+* https://helm.sh/docs/intro/install/[Helm Chart]
+* https://www.okta.com/[Okta OpenID Provider]
+
+== Example Application
+
+We will use a simple web application in this guide that consists of a single https://github.com/wildfly-security-incubator/elytron-examples/blob/main/simple-webapp-okta/src/main/java/org/wildfly/security/examples/SecuredServlet.java[servlet]. We will use the https://github.com/wildfly-security-incubator/elytron-examples/tree/main/simple-webapp-okta[simple-webapp-okta] application in this repository.
+
+We will need to first fork, and then clone the https://github.com/wildfly-security-incubator/elytron-examples[elytron-examples] repository. We will be using the `simple-webapp-okta` directory in this repo:
+
+[source,bash]
+----
+git clone git@github.com:wildfly-security-incubator/elytron-examples.git
+cd simple-webapp-okta
+----
+
+include::{includedir}/_proc-log-into-openshift-cluster.adoc[]
+
+== Configure Okta
+
+For this guide, we will be using Okta as out OpenID provider. In order to secure the application, we will need to access the Okta web console and register our application as a client. 
+
+. Log in to the https://www.okta.com/free-trial/[Okta dashboard] to get started. 
+. At the top right corner, click on *Admin* to be redirected to the admin dashboard. 
+. On the left hand panel, from the *Applications* drop down, click on *Applications* and then *Create App Integration*. Choose *OIDC - OpenID Connect* under *Sign-in method* and *Web Application* for *Application type* and click *Next*. 
```

Comment:
fixed!

Reply:
If possible, it would be good to add links to relevant Okta docs for the different steps (e.g., we did something like that for the "Configure Auth0" section here: https://www.wildfly.org/guides/security-oidc-auth0-openshift).

Reply:
I added [this](https://help.okta.com/en-us/content/topics/apps/apps-configure-settings.htm) for adding an app integration and [this](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-people.htm) for user management for this section. (although they are not documentation, more like step by step guides)
Is there another topic I should add docs for in this section? Or did you want me to replace those with documentation from Okta?

Reply:
Oh, I see it now. :) I think I was expecting it to be at the end of this bullet line but now realize that the next bullet is a continuation and it appears there.